pkg/chunk/chunk_store_test.go

```diff
@@ -65,35 +65,40 @@ var stores = []struct {
 }
 
 // newTestStore creates a new Store for testing.
-func newTestChunkStore(t *testing.T, schemaName string) Store {
+func newTestChunkStore(schemaName string) (Store, error) {
 	var storeCfg StoreConfig
 	flagext.DefaultValues(&storeCfg)
-	return newTestChunkStoreConfig(t, schemaName, storeCfg)
+	return newTestChunkStoreConfig(schemaName, storeCfg)
 }
 
-func newTestChunkStoreConfig(t *testing.T, schemaName string, storeCfg StoreConfig) Store {
+func newTestChunkStoreConfig(schemaName string, storeCfg StoreConfig) (Store, error) {
```

Comment:
Instead of littering this function with `if err != nil ...`, could you change `t` to a `require.TestingT`? https://godoc.org/github.com/stretchr/testify/require#TestingT

Reply:
(That would allow this function to be used in tests and benchmarks)